template/package.json

```diff
@@ -5,13 +5,14 @@
   "scripts": {
     "android": "react-native run-android",
     "ios": "react-native run-ios",
+    "macos": "react-native run-macos",
     "start": "react-native start",
     "test": "jest",
     "lint": "eslint ."
   },
   "dependencies": {
     "react": "17.0.2",
-    "react-native": "0.66.0"
+    "react-native-macos": "1000.0.0"
```

Comment:
should this be both react-native and react-native-macos on version `1000.0.0`?

Reply:
This template is used in E2E CircleCI tests, where the test packs react-native-macos locally into a tar.g.z, makes a test app using this template, then unpacks that tar as the `react-native-macos` dependency. Upstream, the test does thes same thing, except packing `react-native` instead of `react-native-macos`. 

https://github.com/microsoft/react-native-macos/blob/cfdd349903a1960b8fd3ae27fd0e8d000bf82e77/scripts/run-ci-e2e-tests.js#L84

If I specify `react-native` v1000, the test will fail as it doesn't know where to find that package (it's not packed as a tar, and it's not available in NPM). This was the middle ground I found. I originally made this PR to try and fix #949 but there are more errors after this change unfortunately :/ .